tests/test_cli.py

```diff
@@ -36,6 +36,7 @@ def test_that_stripping_works(self):
 		self.assertEqual(stripper('Seasons (with Sjava & Reason)', 'Mozzy'), 'Mozzy-Sjava-and-Reason-Seasons')
 		self.assertEqual(stripper(
 			'거품 안 넘치게 따라줘 [Life Is Good] (feat. Crush, Dj Friz)', 'Dynamic Duo'), 'Dynamic-Duo-Life-Is-Good')
+		self.assertEqual(stripper('ハゼ馳せる果てるまで', 'ZUTOMAYO'), None)
```

Comment:
Actually, you should test the response of lyrics and get_lyrics when a song, artist pair is either song in non-latin, or artist in non-latin or both.